scripts/fixtures/vote-account.json

```diff
@@ -2,11 +2,11 @@
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAfAAAAAAAAAAFAAAAAAAAAANIBAAAAAAAAqDwYBAAAAADBkRIEAAAAANMBAAAAAAAAVuUdBAAAAACoPBgEAAAAANQBAAAAAAAAXG0jBAAAAABW5R0EAAAAANUBAAAAAAAA0/goBAAAAABcbSMEAAAAANYBAAAAAAAAYI0uBAAAAADT+CgEAAAAANcBAAAAAAAAjMMzBAAAAABgjS4EAAAAANgBAAAAAAAAHws6BAAAAACMwzMEAAAAANkBAAAAAAAAYqJABAAAAAAfCzoEAAAAANoBAAAAAAAA7yRHBAAAAABiokAEAAAAANsBAAAAAAAA0atNBAAAAADvJEcEAAAAANwBAAAAAAAAv/FTBAAAAADRq00EAAAAAN0BAAAAAAAA3XdaBAAAAAC/8VMEAAAAAN4BAAAAAAAAIQhhBAAAAADdd1oEAAAAAN8BAAAAAAAAQp9nBAAAAAAhCGEEAAAAAOABAAAAAAAAkjZuBAAAAABCn2cEAAAAAOEBAAAAAAAAXMx0BAAAAACSNm4EAAAAAOIBAAAAAAAAR1N7BAAAAABczHQEAAAAAOMBAAAAAAAAoOiBBAAAAABHU3sEAAAAAOQBAAAAAAAA632IBAAAAACg6IEEAAAAAOUBAAAAAAAAnROPBAAAAADrfYgEAAAAAOYBAAAAAAAAPamVBAAAAACdE48EAAAAAOcBAAAAAAAAnD6cBAAAAAA9qZUEAAAAAOgBAAAAAAAAR9SiBAAAAACcPpwEAAAAAOkBAAAAAAAAF2qpBAAAAABH1KIEAAAAAOoBAAAAAAAAjv+vBAAAAAAXaqkEAAAAAOsBAAAAAAAAZZW2BAAAAACO/68EAAAAAOwBAAAAAAAAvCq9BAAAAABllbYEAAAAAO0BAAAAAAAAb8DDBAAAAAC8Kr0EAAAAAO4BAAAAAAAA2VXKBAAAAABvwMMEAAAAAO8BAAAAAAAAfuvQBAAAAADZVcoEAAAAAPABAAAAAAAAAoDXBAAAAAB+69AEAAAAAPEBAAAAAAAAPRXeBAAAAAACgNcEAAAAAPIBAAAAAAAADKvkBAAAAAA9Fd4EAAAAAPMBAAAAAAAAdkDrBAAAAAAMq+QEAAAAAPQBAAAAAAAAgMDxBAAAAAB2QOsEAAAAAPUBAAAAAAAA61X4BAAAAACAwPEEAAAAAPYBAAAAAAAAZeP+BAAAAADrVfgEAAAAAPcBAAAAAAAAKnkFBQAAAABl4/4EAAAAAPgBAAAAAAAA6A4MBQAAAAAqeQUFAAAAAPkBAAAAAAAACpISBQAAAADoDgwFAAAAAPoBAAAAAAAAaicZBQAAAAAKkhIFAAAAAPsBAAAAAAAAKL0fBQAAAABqJxkFAAAAAPwBAAAAAAAAJk0mBQAAAAAovR8FAAAAAP0BAAAAAAAAp+IsBQAAAAAmTSYFAAAAAP4BAAAAAAAAEngzBQAAAACn4iwFAAAAAP8BAAAAAAAAsw06BQAAAAASeDMFAAAAAAACAAAAAAAAZqNABQAAAACzDToFAAAAAAECAAAAAAAA8DhHBQAAAABmo0AFAAAAAAICAAAAAAAAqs5NBQAAAADwOEcFAAAAAAMCAAAAAAAA9mNUBQAAAACqzk0FAAAAAAQCAAAAAAAAeudaBQAAAAD2Y1QFAAAAAAUCAAAAAAAAojphBQAAAAB651oFAAAAAAYCAAAAAAAAZ9BnBQAAAACiOmEFAAAAAAcCAAAAAAAADGZuBQAAAABn0GcFAAAAAAgCAAAAAAAAxPt0BQAAAAAMZm4FAAAAAAkCAAAAAAAAaJF7BQAAAADE+3QFAAAAAAoCAAAAAAAAriaCBQAAAABokXsFAAAAAAsCAAAAAAAAELyIBQAAAACuJoIFAAAAAAwCAAAAAAAAXVGPBQAAAAAQvIgFAAAAAA0CAAAAAAAACeeVBQAAAABdUY8FAAAAAA4CAAAAAAAAjHycBQAAAAAJ55UFAAAAAA8CAAAAAAAAjASjBQAAAACMfJwFAAAAABACAAAAAAAAspmpBQAAAACMBKMFAAAAABECAAAAAAAAvAivBQAAAACymakFAAAAADKApA0AAAAArHitZAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
```

Comment:
Literally just twiddled this manually by adding 31 bytes of zeros at the end. Hope that works out.

Reply:
OK nevermind. I ended up, instead, dumping the validator's autogenerated vote account, which is probably what @mcintyre94 did in the first place.